plugins/node/opentelemetry-plugin-ioredis/src/types.ts

```diff
@@ -29,3 +30,16 @@ export interface IORedisPluginClientTypes {
   // https://github.com/luin/ioredis/blob/master/API.md
   options: ioredisTypes.RedisOptions;
 }
+
+export type DbStatementSerializer = (
+  cmdName: IORedisCommand['name'],
+  cmdArgs: IORedisCommand['args']
+) => string;
+
+/**
+ * Options available for the IORedis Plugin (see [documentation](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-plugin-ioredis#ioredis-plugin-options))
+ */
+export interface IORedisPluginConfig extends PluginConfig {
+  /** Custom serializer function for the db.statement tag */
+  dbStatementSerializer?: DbStatementSerializer;
```

Comment:
please add some example to documentation so it is easy to understand how this should look like without necessity of looking into source code

Reply:
Done. Let me know if it's enough